ipydatagrid/datagrid.py

```diff
@@ -336,14 +348,17 @@ class DataGrid(DOMWidget):
     selection_mode = Enum(
         default_value="none", values=["row", "column", "cell", "none"]
     ).tag(sync=True)
-    selections = List(Dict()).tag(sync=True, **widget_serialization)
```

Comment:
I'm removing the widgets serializer here, we don't need it as it's a simple dictionary?